evaluator/notifications.go

```diff
@@ -81,7 +82,7 @@ func publishNewAdvisoriesNotification(tx *gorm.DB, inventoryID, accountName stri
 
 	msg, err := mqueue.MessageFromJSON(
 		inventoryID,
-		ntf.MakeNotification(inventoryID, accountName, NewAdvisoryEvent, events))
+		ntf.MakeNotification(inventoryID, accountName, orgID, NewAdvisoryEvent, events))
```

Comment:
And once again :) here - `ntf.MakeNotification(inventoryID, event, NewAdvisoryEvent, events)`.

And finally `MakeNotification()` can unfold event.GetAccountName(), event.GetOrgID() and put it into json.